client/components/logged-out-form/link-item.jsx

```diff
@@ -3,23 +3,17 @@
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
 import React from 'react';
 import classnames from 'classnames';
-import { omit } from 'lodash';
-
-export default class extends React.Component {
-	static displayName = 'LoggedOutFormLinkItem';
 
-	static propTypes = {
-		className: PropTypes.string,
-	};
+export default class LoggedOutFormLinkItem extends React.Component {
```

Comment:
Make it pure while we're here? 😄

Reply:
Yeah, I'm inclined to use a function component here 👍

Reply:
✅  52b59c64e5142f2373c4908bb019229d4f70fe9e